projects/reservoir/fv3/era_process.py

```diff
@@ -0,0 +1,180 @@
+import xarray as xr
+import glob
+import numpy as np
+import vcm
+import vcm.catalog
+from vcm.catalog import catalog
+import xarray as xr
+import netCDF4 as nc
+import glob
+import cftime
+import datetime
+import os
+import argparse
+import os
+
+FREGRID_EXAMPLE_SOURCE_DATA='gs://vcm-ml-raw/2020-11-12-gridspec-orography-and-mosaic-data/C48/*.nc'
+
+var_id_mapping={'sst':'sst','temp2m': 't2m','u_wind':'u10','v_wind':'v10'}
+
+
+
+def add_arguments():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--path",help="path to raw data")
+    parser.add_argument("--variables",nargs='+',help="list of variables to process")
+    return parser.parse_args()
+    
+#raw nc input files are expected to be in path+var+'/downloaded/
+def main(args):
+    path = args.path
+    variables = args.variables
+    setup_fregrid()
+    for var in variables:
+        check_time_steps_complete(path, var)
+        weekdays = ['mon','tue','wed','thu','fri','sat','sun']
+        for i, day in enumerate(weekdays):
+            merge_into_file(path,var,i,day)
+            print(day,' is merged.')
+            regridded = regrid_to_cubed_sphere(path,var,day)
+            print(day,' is regridded.')
+            interpolated = interpolate_nans(var,day)
+            print(day,' is interpolated.')
+            masked = mask(interpolated)
+            print(day,' is masked.')
+            masked.to_netcdf(path + var + '/masked_test/'+var+'_'+day+'.nc') 
```

Comment:
I'd recommend making the output path configurable via the CLI args as opposed to automatically saving it in the same parent dir as the raw data.